Document/0x05a-Platform-Overview.md

```diff
@@ -367,12 +367,39 @@ An example Broadcast Receiver declaration with an intent filter in a manifest:
 </receiver>
 ```
 
+The other way is to create the receiver dynamically in code and register it with the `Context.registerReceiver()` method.
+
+An example of registering Broadcast Receiver dynamically:
+
+```Java
+// Define a broadcast receiver
+myReceiver = new BroadcastReceiver() {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        Log.d(TAG, "Intent received by myReceiver");
+    }
+};
+// Define an intent filter with actions that the broadcast receiver listens for
+IntentFilter intentFilter = new IntentFilter();
+intentFilter.addAction("com.owasp.myapplication.MY_ACTION");
+// To register the broadcast receiver
+registerReceiver(myReceiver, intentFilter);
+// To un-register the broadcast receiver
+unregisterReceiver(myReceiver);
+```
+
+Note that Android apps don’t need to be running to receive intents; the system starts apps automatically when a relevant intent is raised.
+
 After receiving an implicit intent, Android will list all apps that have registered a given action in their filters. If more than one app has registered for the same action, Android will prompt the user to select from the list of available apps.
 
 An interesting feature of Broadcast Receivers is that they are assigned a priority; this way, an intent will be delivered to all authorized receivers according to their priority.
 
 A Local Broadcast Manager can be used to make sure intents are received from the internal app only, and any intent from any other app will be discarded. This is very useful for improving security.
 
+###### Implicit Broadcast Receiver Limitiation
+
+According to [Background Execution Limits](https://developer.android.com/about/versions/oreo/background.html#broadcasts), apps that target Android 8.0 (API level 26) or higher can no longer register Broadcast Receivers for implicit broadcasts in their manifest, except for those listed in [Implicit Broadcast Exceptions](https://developer.android.com/guide/components/broadcast-exceptions). The Broadcast Receivers created at runtime by calling `Context.registerReceiver()` are not affected by this limitation.
+

```

Comment:
Maybe add the earlier limitation on android 7 as well? https://developer.android.com/about/versions/nougat/android-7.0-changes.html?source=post_page---------------------------#bg-opt

Reply:
And limitations for android 9 as well: https://developer.android.com/guide/components/broadcasts

Reply:
Description for both added.